src/base/0.0.0/end-no-respawn-fields.py

```diff
@@ -40,10 +41,23 @@ def migrate(cr, version):
     """
     )
 
+    ignored_fields = set()
+
+    for e in os.environ.get("suppress_upgrade_warnings", "").split(","):
+        if e.startswith("field:"):
+            ignored_fields.add(e[6:])
```

Comment:
Use set-comprehension.
```suggestion
    ignored_fields = set(
        e[6:]
        for e in os.environ.get("suppress_upgrade_warnings", "").split(",")  # noqa: SIM112
        if e.startswith("field:")
    )
```